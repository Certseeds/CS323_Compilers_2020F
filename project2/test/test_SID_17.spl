int main() {
    int a, b, c;
    int d =!!!(a = !!(b = !(c = 0))) ;
    return 0;
}