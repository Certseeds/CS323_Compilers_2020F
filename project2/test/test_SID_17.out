Error type 5 at Line 3: unmatching types on both sides of assignment
Error type 5 at Line 3: unmatching types on both sides of assignment
Error type 5 at Line 3: unmatching types on both sides of assignment
